setup.py

```diff
@@ -75,12 +75,15 @@ def get_version(*file_paths):
         'Development Status :: 5 - Production/Stable',
         'Framework :: Django :: 2.2',
```

Comment:
I think we can remove this line. We are not running the tests for this Django version.

https://github.com/eduNEXT/eox-hooks/blob/master/tox.ini

Reply:
Done